### ISSUE TITLE
Unable to buy Sports and Supers

### ISSUE THREAD

**Issue by mmaust005** [HumanTree92/esx_advancedvehicleshop#78]
**Describe the Bug | A Clear & Concise Description of Bug**
Script is working for the most part. Can not purchase any cars from the Supers or Sports categories. You can successfully purchase vehicles from other categories inside the car dealer.

**Questions**
Have you made changes to anything besides the `config.lua`?:  Yes
Have you looked through the Closed Topics?:  Yes
Have you looked through the Wiki?:  Yes
Are you using a Pre-Installed ESX? If Yes who?: No 
Are you using ESX Legacy?:  Yes
Are you using OneSync?:  Yes
Linux or Windows?:  Windows

**Comment by HumanTree92** [comment]
What files did you change outside of the config.lua? Maybe something that you changed outside of the config.lua is causing the problem. Try reverting the changes.

**Comment by mmaust005** [comment]
I copied the vs_cars shop and configured it to a car dealer employee-only shop for a car dealer job. Was wanting to remove some cars from the general purchases that only car dealer employees can purchase for resale. Everything works fine except for the supers and sports cars. 

I have also removed all categories and put all the cars in the same category. Still unable to purchase some cars. All the cars show up, I get now errors while in the vehicle shop. 

I've looked through everything I've modified and its just stumping me like crazy.
